docs/reference/commandline/create.md

```diff
@@ -69,6 +69,7 @@ Creates a new container.
       -P, --publish-all=false       Publish all exposed ports to random ports
       -p, --publish=[]              Publish a container's port(s) to the host
       --pid=""                      PID namespace to use
+      --pids-limit=0                Tune container pids limit (set -1 for unlimited)
```

Comment:
@cyphar agreed,

@diogomonica I ran several programs and tested various containers and I think 512 is a good number... it will be applied to the cgroup parent which means not every container gets 512, they get a portion, java uses quite a lot so we are trying to provide security while not having people just change it to unlimited